src/ray/util/logging_test.cc

```diff
@@ -230,6 +230,27 @@ TEST(LogPerfTest, PerfTest) {
   RayLog::ShutDownRayLog();
 }
 
+TEST(PrintLogTest, TestCheckOp) {
+  int i = 1;
+  RAY_CHECK_EQ(i, 1);
+  ASSERT_DEATH(RAY_CHECK_EQ(i, 2), "1 vs 2");
```

Comment:
Why doesn't this output the `RAY_CHECK` msg? Or maybe also add a test for when both inputs are variables?

Reply:
It does. "1 vs 2" is just substring match.

Reply:
Added the suggested test.